src/can.rs

```diff
@@ -615,6 +615,14 @@ where
         })
     }
 
+    pub fn driver(&self) -> &CanDriver<'d> {
```

Comment:
Looks like these changes snuck in?

Reply:
Yeah. Being lazy over there. We need these changes, but I should've opened a separate PR for these.